src/Flex.php

```diff
@@ -231,6 +232,15 @@ function ($value) {
             }
         }
 
+        if (null !== $manifest) {
+            array_unshift(
+                $this->postInstallOutput,
+                '',
+                '<info>A few files may have been created to configure newly installed packages.</>',
+                'Don\'t hesitate to <comment>review</>, <comment>edit</> and <comment>commit</> them: these files are <comment>*yours*</>.'
```

Comment:
I don't think you should put stars in it. If you want to make it bold, make it bold for real (I think the Console component supports it)

Reply:
The console supports it, but not all terminals 😉

Reply:
stars removed (bold+yellow didn't look nice)